src/remote_interface.c

```diff
@@ -243,11 +243,11 @@ RemoteInterfaceContext RemoteInterface_Context = {
     .description = "Remote interface",
     .num_max_thread = 1024,  // 1 <= num_max_thread <= AMQP_DEFAULT_MAX_CHANNELS
     .num_working_thread = 0,
-    .initialize = PoWRemoteInterface_Context_Initialize,
-    .destroy = PoWRemoteInterface_Context_Destroy,
-    .getPoWContext = PoWRemoteInterface_getPoWContext,
-    .freePoWContext = PoWRemoteInterface_freePoWContext,
+    .initialize = RI_Context_Initialize,
```

Comment:
We might unify the name scheme for consistency. That is, `RI_init`, `RI_destroy`, `RI_getPoWContext`, `RI_freePoWContext`, `RI_doPoW`, etc.